mantis-examples/mantis-examples-mantis-publish-sample/build.gradle

```diff
@@ -34,7 +34,7 @@ buildscript {
         jcenter()
     }
     dependencies {
-        classpath 'com.github.jengelman.gradle.plugins:shadow:4.0.4'
+        classpath 'com.github.jengelman.gradle.plugins:shadow:6.1.0'
```

Comment:
@rpalcolea any breaking changes between these versions which would prevent shading? Asking to see if we need to change any other gradle configs.

Reply:
I don't think so. It is more about compatibility with modern versions of Gradle other than breaking changes. We upgraded multiple projects/apps without issues

The main breaking changes in shadow plugin came from 2.x to 4.x but not your case